### PR TITLE
Tightened beta.md text

### PR DESCRIPTION
**Pull request by tyg**

### CODE REVIEW

File: main/getting-started/beta.md [hunk 1]
```diff
@@ -55,10 +66,12 @@ const proposal = {
...
 ```
 
-This deadline is the expiration date for the covered call option. If the option has not been exercised by the deadline, as specified in the above code example, its underlying assets are paid out to the contract's creator as a refund of their escrowed assets.
+The *deadline* is the option's expiration date. The above code specifies that if the option 
+has not been exercised by the deadline, its underlying assets are paid to the contract's creator 
+as a refund of their escrowed assets.
 
 ## Developer Experience
-Agoric is a team of engineers. We've based our developer experience on our decades of open-source software experience. In our beta, you can expect the following:
+Agoric is a team of engineers. We've based our developer experience on our decades of open-source software experience. In our alpha, you can expect the following:
```
Comment (katelynsills) [inline main/getting-started/beta.md:74]: this is a regression.

File: main/getting-started/beta.md [hunk 2]
```diff
@@ -3,27 +3,32 @@ Release Version: Baltic
 
 ![Agoric Banner](./assets/agoric-banner-2.png)
 
-Welcome to Phase 1 of the Agoric Beta!
+Welcome to the Agoric Beta!
```
Comment (katelynsills) [inline main/getting-started/beta.md:6]: We need the Phase 1 phrasing

**Review by katelynsills — CHANGES_REQUESTED** [review]
I think two things are worse. Let's put this aside for now, thank you for making the pull request, though

**Review by rowgraus — APPROVED** [review]
LGTM

**Pull request merged**